modules/openapi-generator/src/main/resources/Java/libraries/jersey3/JSON.mustache

```diff
@@ -30,7 +30,7 @@ public class JSON implements ContextResolver<ObjectMapper> {
     mapper = JsonMapper.builder()
             .serializationInclusion(JsonInclude.Include.NON_NULL)
             .configure(MapperFeature.ALLOW_COERCION_OF_SCALARS, false)
-            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, true)
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, {{failOnUnknownProperties}})
```

Comment:
are there plans to have something similar for Kotlin generators too? I got exactly the same issue using Kotlin + okhttp3 client and seems like [there](https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/kotlin-client/jvm-common/infrastructure/Serializer.kt.mustache#L113) we use the same properties and in Java's generators.

Reply:
I can probably take a look once this PR is merged. Is there a issue already?

Reply:
thanks a lot. I am not aware of formal issue yet but please let me know if I should open one.

Reply:
Yes, that would be great!

Reply:
here it is https://github.com/OpenAPITools/openapi-generator/issues/19408 . Thanks!